test/support/pulp/task_support.rb

```diff
@@ -23,21 +23,5 @@ def self.wait_on_tasks(task_list, options = {})
       puts e
       puts e.backtrace
     end
-
-    module TaskRunningWithVcr
-      extend ActiveSupport::Concern
-
-      included do
-        singleton_class.alias_method_chain :any_task_running, :vcr
-      end
-
-      module ClassMethods
-        def any_task_running_with_vcr(async_tasks)
-          VCR.live? ? any_task_running_without_vcr(async_tasks) : false
-        end
-      end
-    end
-
-    Katello::PulpTaskStatus.send(:include, TaskRunningWithVcr)
```

Comment:
I get what's happening in this PR, but this change breaks the pattern. I guess I'm not sure why we need to remove this. Could you please explain?